### DISCUSSION TITLE
"Response body parse failed. See previous exception for details."

### DISCUSSION THREAD

**Discussion by MusicStudioNYC** [tectalichq/public-openai-client-php#26] [General] [answered]
Didn't find the place to report "Issues" so I'm writing this here.

I get this error if the completion begins with a stop token.
I think this library should be able to understand that scenario and provide the appropriate error message, not just end up in a generic "Parse failed"

**Comment by thejamescollins** [comment, marked as answer]
Hi there @UpTheDesign,

To help us investigate this, could you please reply with the full code and request that you are sending to OpenAI (excluding your authentication token of course).

Thank you,

James
